app/lib/discourse_automation/scriptable.rb

```diff
@@ -125,10 +125,38 @@ def utils
     end
 
     module Utils
+      def self.fetch_report(name, args = {})
+        report = Report.find(name, args)
+
+        return nil if !report
+
+        return if !report.modes.include?(:table)
+
+        ordered_columns = report.labels.map { |l| l[:property] }
+
+        table = "\n"
+        table += '|' + report.labels.map { |l| l[:title] }.join('|') + "|\n"
+        table += '|' + report.labels.count.times.map { '-' }.join('|') + "|\n"
+        report.data.each do |data|
+          table += '|'
+          table += ordered_columns.map { |col| data[col] }.join('|')
+          table += "|\n"
```

Comment:
Someone fairly recently did a benchmark and if I remember correctly, `<<` is faster?

Reply:
it doesn't work with frozen string though no ?

Reply:
This would work though, but I'll leave it up to you 🙂 

```ruby
        table = +"\n"
        table << "|#{report.labels.map { |l| l[:title] }.join('|')}|\n"
        table << "|#{report.labels.count.times.map { '-' }.join('|')}|\n"
        report.data.each do |data|
          table << "|#{ordered_columns.map { |col| data[col] }.join('|')}|\n"
```

Reply:
ok fine by me will make the change :+1: